pkg/vere/io/http.c

```diff
@@ -546,6 +553,57 @@ _http_req_dispatch(u3_hreq* req_u, u3_noun req)
   }
 }
 
+
+/* _http_cache_scry_cb(): insert scry result into noun cache
+*/
+static void
+_http_cache_scry_cb(void* vod_p, u3_noun nun)
+{
+  u3_httd* htd_u = vod_p;
+
+  // XX need both the scry path and the noun
+  u3_noun pax = XX;
```

Comment:
What's the proper way to pass both a `u3_httd*` and a scry path to this callback? Should I define a struct for this, or can I get away with something ad-hoc declared inline?

Reply:
yeah sounds like you need a struct